partials/comparison-cards.html

```diff
@@ -1,9 +1,11 @@
-{% if page.track == 'javascript' %}
-  {% assign track = "JavaScript" %}
+{% if page.track == 'JavaScript' %}
```

Comment:
Ideally this information would be passed in, the partials shouldn't have any knowledge of JavaScript vs C#. We are going to eventually create a Java page, we should only have to add a java.html YAML file